ndic/search.py

```diff
@@ -28,3 +30,50 @@ def search(search_word, xth=1):
     response = request_naver_endic_url(naver_endic_url)
     word_meaning = get_word_meaning(response, xth)
     return word_meaning
+
+
+def search_zh_cli(search_word, num=1):
+    """
+    Search chinese-zh and return result converted to string
+    for cli environment
+
+    :param search_word: chinese or korean word
+    :param num: number of result
+    :return:
+    """
+
+    if num < 1:
+        raise ValueError("search_zh_cli() got parameter res_len under 1")
+
+    naver_zhdic_url = make_naver_zhdic_url(search_word)
+    response_json = request_naver_zhdic_url(naver_zhdic_url)
+    parsed_items = parse_zh_json(response_json, num)
```

Comment:
How about calling the `search_zh(search_word, num)` instead of the 3 lines above?
These logics are duplicated.